modules/migration-manual-rollback.adoc

```diff
@@ -0,0 +1,90 @@
+// Module included in the following assemblies:
+// migration/migrating-3-4/troubleshooting.adoc
+// migration/migrating-4-4/troubleshooting.adoc
+// migration/migrating-4-4/troubleshooting.adoc
+[id='migration-manual-rollback_{context}']
+= Manual rollback
```

Comment:
Procedures should start with gerunds, so I'd title it "Manually rolling back a migration"